### DISCUSSION TITLE
iOS TestFlight apps

### DISCUSSION THREAD

**Discussion by Tomsgurl4** [pluwen/awesome-testflight-link#223] [Q&A]
What are good sites and apps that are beta testing and go through TestFlight

**Comment by justg5594** [comment]
Departures.com

**Reply by hucynogen82** [reply]
H

**Reply by Tomsgurl4** [reply]
Is there any other pages or sites

**Comment by artyomsklyarov** [comment]
How do I submit app to this list?

**Reply by pluwen** [reply]
create an issue
